src/status_im2/contexts/quo_preview/profile/profile_card.cljs

```diff
@@ -0,0 +1,90 @@
+(ns status-im2.contexts.quo-preview.profile.profile-card
+  (:require [quo2.foundations.colors :as colors]
+            [react-native.core :as rn]
+            [reagent.core :as reagent]
+            [quo2.core :as quo]
+            [status-im.react-native.resources :as resources]
+            [status-im2.contexts.quo-preview.preview :as preview]))
+
+(def descriptor
+  [{:label "Show: Sign in to this profile?"
+    :key   :show-sign-profile?
+    :type  :boolean}
+   {:label "Show: Is from key card?"
+    :key   :key-card?
+    :type  :boolean}
+   {:label "Show: Emoji hash?"
+    :key   :show-emoji-hash?
+    :type  :boolean}
+   {:label   "Customization Color"
+    :key     :customization-color
+    :type    :select
+    :options [{:key   :primary
+               :value "Primary"}
+              {:key   :purple
+               :value "Purple"}
+              {:key   :indigo
+               :value "Indigo"}
+              {:key   :turquoise
+               :value "Turquoise"}
+              {:key   :blue
+               :value "Blue"}
+              {:key   :green
+               :value "Green"}
+              {:key   :yellow
+               :value "Yellow"}
+              {:key   :orange
+               :value "Orange"}
+              {:key   :red
+               :value "Red"}
+              {:key   :pink
+               :value "Pink"}
+              {:key   :brown
+               :value "Brown"}
+              {:key   :beige
+               :value "Beige"}]}
+   {:label "Hash"
+    :key   :hash
+    :type  :text}
+   {:label "Name"
+    :key   :name
+    :type  :text}
+   {:label "Emoji hash"
+    :key   :emoji-hash
+    :type  :text}])
+
+(defn cool-preview
+  []
+  (let [state (reagent/atom {:show-sign-profile? true
+                             :key-card? true
+                             :name "Matt Grote"
+                             :on-press-dots nil
+                             :on-press-sign nil
```

Comment:
so here will be something like `:sign-label (i18n/label :t/sign-in-profile)`

Reply:
Yes I asked about that previously but this is what @ilmotta said to me: https://github.com/status-im/status-mobile/pull/14652#discussion_r1058292032

Reply:
What should I do?